src/msg/Message.h

```diff
@@ -546,7 +546,7 @@ class SafeMessage : public Message {
 
 namespace ceph {
 template<class T, typename... Args>
-ceph::ref_t<T> make_message(Args&&... args) {
+MRef<T> make_message(Args&&... args) {
```

Comment:
i'd prefer using `ceph::ref_t`. there is not need to have two groups of aliases mirroring each other in the same source tree just for emphasizing that one is dedicated to `Message` while another is for all types in general.

Reply:
@tchaikov I'll abandon this commit.